README.md

```diff
@@ -1,207 +1,225 @@
+| Spring Data | 1.8.0+ | `spring-data` | |
+| Spring Web (MVC) | 4.0+ | `spring-web` | Includes DispatcherServlet and HandlerAdapter. Supports exceptions whitelist using `@TraceSetting`. |
+| Spring WebFlux | 5.0.0+ | `spring-webflux` | |
+| Vertx Web | 3.0.0+  | `vertx` | This works through the Netty instrumentation for requests, and also includes spans for handlers. |
 
-_Italicized_ libraries are in beta and must be explicitly enabled by setting the
-`-Dsignalfx.integration.<name>.enabled=true` system property, where `<name>` is
-the name specified in the table.
+## Configure the SignalFx Java Agent
 
-## Configuration and Usage
+Send traces from your Java application to a local or remote Smart Agent,
+OpenTelemetry Collector, or SignalFx ingest endpoint.
 
-As demonstrated above, using the Java Agent requires downloading a recent
-release JAR and specifying its path as the value for the `javaagent` JVM startup
-option. Depending on your workflow, this should be done via the command line or
-the appropriate configuration mechanism for your project management system.
+### Configuration values
 
-**Note: Although several `javaagent` options can be specified for a given Java
-application, we cannot confirm support for using multiple agents, especially
-those that provide similar instrumentation capabilities, as issues arising from
-conflicting bytecode modifications are possible.**
+The agent needs the following properties or environment variables for configuring
+tracer functionality and trace content. System property values take priority
+over corresponding environment variables.
 
-The SignalFx Java Agent uses a few properties or environment variables for
-configuring tracer functionality and [trace
-content](https://docs.signalfx.com/en/latest/apm/apm-overview/apm-metadata.html):
-
-| System Property | Environment Variable | Default Value | Notes |
+| System property | Environment variable | Default value | Notes |
 | ---             | ---                  | ---           | ---   |
-| `signalfx.service.name` | `SIGNALFX_SERVICE_NAME` | `"unnamed-java-app"` | |
-| `signalfx.agent.host` | `SIGNALFX_AGENT_HOST` | `"localhost"` | |
-| `signalfx.endpoint.url` | `SIGNALFX_ENDPOINT_URL` | `"http://localhost:9080/v1/trace"` | Takes priority over constituent Agent properties. |
-| `signalfx.tracing.enabled` | `SIGNALFX_TRACING_ENABLED` | `"true"` | Globally enables tracer creation and auto-instrumentation.  Any value not matching `"true"` will be treated as false (`Boolean.valueOf()`). |
+| `signalfx.service.name` | `SIGNALFX_SERVICE_NAME` | `"unnamed-java-app"` | The name of the service. |
+| `signalfx.agent.host` | `SIGNALFX_AGENT_HOST` | `"localhost"` | The endpoint for a SignalFx Smart Agent or OpenTelemetry Collector. |
+| `signalfx.endpoint.url` | `SIGNALFX_ENDPOINT_URL` | `"http://localhost:9080/v1/trace"` | Takes priority over constituent Agent properties. If you're sending traces directly to SignalFx, specify the SignalFx endpoint here. |
```

Comment:
again, not supported atm

Reply:
just to be clear, remove `SIGNALFX_ENDPOINT_URL`?

Reply:
sending to ingest